modules/apps/journal/journal-service/src/main/java/com/liferay/journal/internal/util/JournalConverterImpl.java

```diff
@@ -164,7 +164,7 @@ private void _addDDMFields(
 
 				_updateFieldsDisplay(
 					ddmFields, ddmFormField.getName(),
-					String.valueOf(ddmStructure.getStructureId()));
```

Comment:
@Mikellorza, change looks good, just a thing. If you go to the _updateFieldsDisplay, you will see

```
if (Validator.isNull(instanceId)) {
	instanceId = StringUtil.randomString();
}
```
so wouldn't it be better to remove the parameter from the method altogether and let instanceId always be a random string?

Reply:
Forget about it, IJ played tricks on me